bindings/yarp.i

```diff
@@ -463,7 +464,10 @@ MAKE_COMMS(Bottle)
   %template(DVector) std::vector<double>;
   %template(BVector) std::vector<bool>;
   %template(SVector) std::vector<std::string>;
+  // Extend IVector for handling conversion of vectors from and to Matlab
+  %include "IVector_fromTo_matlab.i"
```

Comment:
This should be included only when generating the code for matlab (mex-based) bindings, so perhaps it should be guarded by an `#ifdef SWIGMATLAB` clause? I wonder how it is compiling fine on Travis.